Sources/performance-worker/BenchmarkClient.swift

```diff
@@ -80,19 +103,138 @@ struct BenchmarkClient {
     }
   }
 
+  private func timeIt<R>(
+    _ body: () async throws -> R
+  ) async rethrows -> (R, nanoseconds: Double) {
+    let startTime = DispatchTime.now().uptimeNanoseconds
+    let result = try await body()
+    let endTime = DispatchTime.now().uptimeNanoseconds
+    return (result, nanoseconds: Double(endTime - startTime))
+  }
+
   // The result is the number of nanoseconds for processing the RPC.
   private func makeRPC(
-    client: Grpc_Testing_BenchmarkServiceClient,
-    rpcType: Grpc_Testing_RpcType
-  ) -> (latency: Double, errorCode: RPCError.Code?) {
+    benchmarkClient: Grpc_Testing_BenchmarkServiceClient,
+    rpcType: RpcType
+  ) async throws -> (latency: Double, errorCode: RPCError.Code?) {
+    let request = Grpc_Testing_SimpleRequest.with {
```

Comment:
Can we call this `message`? I think of the "request" as the overall input to the RPC, i.e. it includes the metadata as well. A "message" or "messages" are a part of the request.